XCode/Sources/Files.swift

```diff
@@ -37,7 +37,19 @@ public func shareFilesFromDirectory(_ directoryPath: String, defaults: [String]
         if let file = try? (directoryPath + String.pathSeparator + fileRelativePath.value).openForReading() {
             let mimeType = fileRelativePath.value.mimeType()
             
-            return .raw(200, "OK", ["Content-Type": mimeType], { writer in
+            var responseHeader: [String: String]
```

Comment:
This looks good @nichbar !. What do you think if we clean it a little as we are not using the error at all of the `attributesOfItem(_:)` and at the same time we avoid the force unwrap reported by Swiftlint? 

We can create the `filePath` variable at the top as it would be used later inside the `if` statement too and we can use optional binding instead of the normal `do-catch` dance as we're not using or propagating the error outside the function. 

I think something like this would work:

```swift
let filePath = directoryPath + String.pathSeparator + fileRelativePath.value
        
if let file = try? filePath.openForReading() {
    let mimeType = fileRelativePath.value.mimeType()
    var responseHeader: [String: String] = ["Content-Type": mimeType]
    
    if let attr = try? FileManager.default.attributesOfItem(atPath: filePath),
        let fileSize = attr[FileAttributeKey.size] as? UInt64 {
        responseHeader["Content-Length"] = String(fileSize)
    }
    
    return .raw(200, "OK", responseHeader, { writer in
        try? writer.write(file)
        file.close()
    })
}
```

Let me know your thoughts 🙂.

Reply:
@Vkt0r Thanks for pointing it out!

Your modification make these code way more readable. 

Should i rebase my branch or create another PR to submit your modification?

Reply:
Oh, no need to create another PR, please modify your current PR and squash it with your first commit and we're ready to go.